app/controllers/merchant/orders_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Merchant::OrdersController < ApplicationController
+
+  def show
+    @order = Order.find(params[:order_id])
+    @merchant = Merchant.find(current_user[:merchant_id])
```

Comment:
We are not supposed to send 2 variables to the view, but I couldn't figure out how to use `current_user.merchant_id` to get the merchant within the view to display only their items